hq_superset/utils.py

```diff
@@ -37,3 +47,51 @@ def get_role_name_for_domain(domain):
     # Prefix in-case domain name matches with known role names such as admin
     # Same prefix pattern as schema only by coincidence, not a must.
     return f"{DOMAIN_PREFIX}{domain}"
+
+
+def get_column_dtypes(datasource_defn):
+    """
+    Maps UCR column data types to Pandas data types.
+
+    See corehq/apps/userreports/datatypes.py for possible data types.
+    """
+    # TODO: How are array indicators handled in CSV export?
+    pandas_dtypes = {
+        'date': 'datetime64[ns]',
```

Comment:
I am curious if where the format of the values here come from. Per pandas docs [here](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_sql.html) this should be SQLAlchemy datatype?

Reply:
These values are being passed to [`read_csv()`](https://github.com/dimagi/hq_superset/pull/13/commits/63c22451dbb446500f325da37a65e5013eae4d23#diff-904cccd73cc4407182ae5580289ad118ed1be682677edf1c7e0ba03489788699R129), when the Dataframe is being created, not to `to_sql()`. `to_sql()` respects the Dataframe's data types.